### ISSUE TITLE
reaction_api_1 crashes constantly

### ISSUE THREAD

**Issue by hslaa** [reactioncommerce/reaction-platform#162]
Are you looking for help with getting started on Reaction? Please visit our [Reaction documentation](https://docs.reactioncommerce.com/reaction-docs/master/getting-started-developing-with-reaction).

## Prerequisites
* [X] Are you running the latest version?
* [X] Are you able to consistently reproduce the issue?
* [X] Did you search the issue queue for existing issue? [Search issues](https://github.com/reactioncommerce/reaction/issues)

## Issue Description
Everytime I try to upload a product image in the reaction admin, the container `reaction_api_1` crashes with message:

```log
TypeError: Cannot read property 'message' of undefined

at GridFSStore.<anonymous> (/usr/local/src/app/node_modules/@reactioncommerce/file-collections/dist/common/FileCollection.js:85:78)

at GridFSStore.emit (events.js:314:20)

at GridFSStore.EventEmitter.emit (domain.js:486:12)

at GridWriteStream.emitError (/usr/local/src/app/node_modules/@reactioncommerce/file-collections-sa-base/dist/StorageAdapter.js:159:28)

at GridWriteStream.emit (events.js:326:22)

at GridWriteStream.EventEmitter.emit (domain.js:486:12)

at GridWriteStream.FlushWritable.emit (/usr/local/src/app/node_modules/flushwritable/lib/FlushWritable.js:37:31)

at GridWriteStream._error (/usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:246:7)

at /usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:265:8

at processTicksAndRejections (internal/process/task_queues.js:75:11)

09:57:56.829Z INFO Reaction: Stopping Reaction API...

(node:18) DeprecationWarning: collection.save is deprecated. Use insertOne, insertMany, updateOne, or updateMany instead.

(node:18) DeprecationWarning: collection.update is deprecated. Use updateOne, updateMany, or bulkWrite instead.

npm ERR! code ELIFECYCLE

npm ERR! errno 1

npm ERR! reaction-api@3.10.0 start: node --experimental-modules --experimental-json-modules ./src/index.js

npm ERR! Exit status 1



npm ERR!

npm ERR! Failed at the reaction-api@3.10.0 start script.

npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:

npm ERR! /home/node/.npm/_logs/2020-10-14T09_57_56_886Z-debug.log 
```

## Steps to Reproduce
Please provide starting context, i.e. logged in as a user, configure a particular payment method.
1. Installed the platform as instructed _today_ (That would mean I have the latest version, right?)
2. Created a user in the admin web interface, logged in
3. Created a product with two variants.
4. Try to upload an image to either product or variants
5. The `reaction_api_1` container crashes with error above.

**Comment by ghost** [comment]
@hslaa If you're on API version 3.10 that was just released a day or two ago, yeah you're on the latest. I haven't upgraded yet but if you're stuck you could try backing down to 3.9 to determine if you're experiencing the same issue and unblock yourself.

**Comment by creative-ae** [comment]
im facing same issue.. i'm using the latest. 
any solution ?

**Comment by phantom-limb** [comment]
I have the same problem in reactioncommerce/reaction:3.10.0 
but I see more when I try to upload some files.

Bugtrace:
api_1    | (node:18) DeprecationWarning: GridStore is deprecated, and will be removed in a future version. Please use GridFSBucket instead
api_1    | (node:18) DeprecationWarning: collection.ensureIndex is deprecated. Use createIndexes instead.
api_1    | TypeError: Cannot read property 'message' of undefined
api_1    |     at GridFSStore.<anonymous> (/usr/local/src/app/node_modules/@reactioncommerce/file-collections/dist/common/FileCollection.js:85:78)
api_1    |     at GridFSStore.emit (events.js:314:20)
api_1    |     at GridFSStore.EventEmitter.emit (domain.js:486:12)
api_1    |     at GridWriteStream.emitError (/usr/local/src/app/node_modules/@reactioncommerce/file-collections-sa-base/dist/StorageAdapter.js:159:28)
api_1    |     at GridWriteStream.emit (events.js:326:22)
api_1    |     at GridWriteStream.EventEmitter.emit (domain.js:486:12)
api_1    |     at GridWriteStream.FlushWritable.emit (/usr/local/src/app/node_modules/flushwritable/lib/FlushWritable.js:37:31)
api_1    |     at GridWriteStream._error (/usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:246:7)
api_1    |     at /usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:265:8
api_1    |     at processTicksAndRejections (internal/process/task_queues.js:75:11)
api_1    | 14:35:51.154Z  INFO Reaction: Stopping Reaction API...
api_1    | (node:18) DeprecationWarning: collection.save is deprecated. Use insertOne, insertMany, updateOne, or updateMany instead.
api_1    | (node:18) DeprecationWarning: collection.update is deprecated. Use updateOne, updateMany, or bulkWrite instead.
api_1    | TypeError: Cannot read property 'message' of undefined
api_1    |     at GridFSStore.<anonymous> (/usr/local/src/app/node_modules/@reactioncommerce/file-collections/dist/common/FileCollection.js:85:78)
api_1    |     at GridFSStore.emit (events.js:314:20)
api_1    |     at GridFSStore.EventEmitter.emit (domain.js:486:12)
api_1    |     at GridWriteStream.emitError (/usr/local/src/app/node_modules/@reactioncommerce/file-collections-sa-base/dist/StorageAdapter.js:159:28)
api_1    |     at GridWriteStream.emit (events.js:326:22)
api_1    |     at GridWriteStream.EventEmitter.emit (domain.js:486:12)
api_1    |     at GridWriteStream.FlushWritable.emit (/usr/local/src/app/node_modules/flushwritable/lib/FlushWritable.js:37:31)
api_1    |     at GridWriteStream._error (/usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:246:7)
api_1    |     at /usr/local/src/app/node_modules/gridfs-stream/lib/writestream.js:265:8
api_1    |     at processTicksAndRejections (internal/process/task_queues.js:75:11)
api_1    | npm ERR! code ELIFECYCLE
api_1    | npm ERR! errno 1
api_1    | npm ERR! reaction-api@3.10.0 start: `node --experimental-modules --experimental-json-modules ./src/index.js`
api_1    | npm ERR! Exit status 1
api_1    | npm ERR! 
api_1    | npm ERR! Failed at the reaction-api@3.10.0 start script.
api_1    | npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
api_1    | 
api_1    | npm ERR! A complete log of this run can be found in:
api_1    | npm ERR!     /home/node/.npm/_logs/2020-11-03T14_35_51_272Z-debug.log
reaction_api_1 exited with code 1

Does anyone have any idea how to fix this?

**Comment by alagiz** [comment]
having the same issue with the latest version, any update on that?

**Comment by nnnnat** [comment]
This issue looks like a duplicate of #6298 in the reaction repo. We're working on a solution now
https://github.com/reactioncommerce/reaction/issues/6298
